### ISSUE TITLE
Register datasets with OBIS (in addition to GBIF)?

### ISSUE THREAD

**Issue by peterdesmet** [inbo/bird-tracking#187]
Here's an overview of dataset that might be applicable for registration with OBIS (in addition to GBIF):

- [x] ~BOP_RODENT~: no, mainly terrestrial
- [x] CURLEW_VLAANDEREN: yes, crossing Bay of Biscay
- [x] ~DELTATRACK~: yes, but not public
- [x] ~H_GRONINGEN~: no, almost exclusively terrestrial
- [x] HG_OOSTENDE: yes
- [x] LBBG_JUVENILE: yes
- [x] LBBG_ZEEBRUGGE: yes
- [x] MEDGULL_ANTWERPEN: yes
- [x] ~MH_ANTWERPEN~: no, almost exclusively terrestrial
- [x] ~MH_WATERLAND~: no, almost exclusively terrestrial
- [x] O_AMELAND: yes, Wadden Sea
- [x] O_ASSEN: yes, Wadden Sea
- [x] O_BALGZAND: yes, IJsselmeer
- [x] O_SCHIERMONNIKOOG: yes, Wadden Sea
- [x] O_VLIELAND: yes, much of North and Baltic Sea
- [x] O_WESTERSCHELDE: maybe, mostly hugging coast

**Comment by peterdesmet** [comment]
@pieterprovoost how do you want me to proceed with these?
- Link one dataset in the IPT to OBIS and see how it goes? I.e. you reporting back?
- Any of the above datasets you consider out of scope?

Note however that the datasets **do not have Aphia ID as `scientificName`**, since they are all transformed with an [R function](https://inbo.github.io/movepub/reference/write_dwc.html) that does not take into account mapping scientific names with Aphia IDs.

**Comment by pieterprovoost** [comment]
@peterdesmet I need to get endorsement from one of our nodes, once that is done I can proceed to harvesting. I agree with the selection above, and I don't expect any issues with taxon matching given the limited taxonomic scope.

**Comment by pieterprovoost** [comment]
I received confirmation that these can be published under EurOBIS. It would be good if you could link them to the OBIS network in IPT already, that will ensure that the statistics on our network page are more or less accurate.

No need to add `scientificNameID`, our automated taxon matching should take care of that. In case of any issues in the future, would you be willing to accept a PR for (optionally) adding WoRMS (or other) identifiers in `movepub`?

One issue we identified is that we currently require every `parentEventID` to have a matching `eventID` within the dataset. For your datasets that would mean that a warning is displayed on the dataset pages in OBIS. However, I don't think this requirement aligns very well with the current definition of `parentEventID`, so I'll try to implement a fix before I start harvesting.

Thanks again!

**Comment by peterdesmet** [comment]
Ok, I have added the [OBIS network](https://www.gbif.org/network/2b7c7b4f-4d4f-40d3-94de-c28b6fa054a6) to 7 datasets in the IPT now (4 additional ones are not yet published). It is unclear to me if that information is available without republishing? See e.g. https://ipt.inbo.be/resource?r=lbbg_juvenile

> In case of any issues in the future, would you be willing to accept a PR for (optionally) adding WoRMS (or other) identifiers in `movepub`?

Sure!

> every `parentEventID` to have a matching `eventID`

Indeed, that won't be the case for these dataset, after careful consideration 😄  (see https://github.com/inbo/movepub/issues/10)

**Comment by pieterprovoost** [comment]
`CURLEW_VLAANDEREN` was added successfully, if you agree I'll add the others as well. See https://obis.org/dataset/7ee5747e-f7c5-44ad-9012-925dd60967aa

The network link does not require republishing, the change is made instantly through the GBIF registry API.

**Comment by peterdesmet** [comment]
Cool! Looks good, you can add the others.

I assume these datasets won't be pushed (and duplicated) to GBIF?

**Comment by peterdesmet** [comment]
- DELTATRACK: won't be public, should not have included it it my list
- HG_OOSTENDE, O_AMELAND, O_BALGZAND, O_SCHIERMONNIKOOG: not yet published, but those will indeed be the final shortnames (lowercase). Once published, I will also link them with the OBIS network in the IPT.
- Note that BOP_RODENT, H_GRONINGEN , MH_ANTWERPEN, MH_WATERLAND should not be harvested as they are not marine.

**Comment by peterdesmet** [comment]
O_BALGZAND and O_SCHIERMONNIKOOG are now also published. HG_OOSTENDE and O_AMELAND are pending more info from the researchers.

**Comment by pieterprovoost** [comment]
| OBIS dataset | IPT URL |
|---|---|
| https://obis.org/dataset/2c6aa97e-e886-4564-a55a-48e2e506f014 | https://ipt.inbo.be/resource?r=o_balgzand |
| https://obis.org/dataset/01dbc62a-e166-4752-8547-6db4542ec039 | https://ipt.inbo.be/resource?r=o_schiermonnikoog |

**Comment by peterdesmet** [comment]
@pieterprovoost what data provider is associated with these datasets? E.g. would INBO appear in the "data providers" section at https://obis.org/node/4bf79a01-65a9-4db6-b37b-18434f26ddfc?

**Comment by pieterprovoost** [comment]
We are currently only using affiliation info from the EML, which is not present for these datasets. A quick fix would be to enter an organisation for at least one of the creators, but I'll look into ORCID integration as well.

**Comment by peterdesmet** [comment]
I noticed in the EML specs that `organisation` is actually reserved for when the agent is not a person and that it therefore shouldn't be used for affiliation. That is why I didn't map that as such in my movepub function, but many people of course do use it for affiliation (myself included for other datasets).

**Comment by pieterprovoost** [comment]
@peterdesmet I have added (basic) ORCID integration to our metadata parser, see the data providers panel at https://obis.org/dataset/550b4cc1-c40d-4070-a0cb-26e010eca9d4 If no organization has been provided in the EML we'll try to get one from ORCID.

I'll reprocess the other datasets as well.

**Comment by peterdesmet** [comment]
👏 such cool features!

**Comment by peterdesmet** [comment]
@pieterprovoost HG_OOSTENDE is now published

**Comment by pieterprovoost** [comment]
Thanks! https://obis.org/dataset/00cad65a-aa33-4d98-93a2-15155fa963e3

**Comment by peterdesmet** [comment]
And finally, the last one: `O_AMELAND` at https://ipt.inbo.be/resource?r=o_ameland

**Comment by pieterprovoost** [comment]
Done https://obis.org/dataset/3b1da04e-7b8d-4080-ba17-d29909d6d95b
